tests/functional/syntax/test_msg_data.py

```diff
@@ -41,7 +41,7 @@ def foo(bar: uint256) -> Bytes[36]:
     contract = get_contract(code)
 
     # 2fbebd38000000000000000000000000000000000000000000000000000000000000002a
-    method_id = keccak(text="foo(uint256)").hex()[2:10]  # 2fbebd38
+    method_id = keccak(b"foo(uint256)").hex()[:8]  # 2fbebd38
```

Comment:
i guess we can use the `method_id()` util here to be consistent with the `test_variable_assignment` above